schema/partner.js

```diff
@@ -70,6 +70,10 @@ const PartnerType = new GraphQLObjectType({
         type: GraphQLBoolean,
         resolve: ({ pre_qualify }) => pre_qualify,
       },
+      is_fair_partner: {
+        type: GraphQLBoolean,
+        resolve: ({ claimed }) => !claimed,
```

Comment:
I was going to comment in the other PR, but this kinda seems roundabout and confusing to use `claimed`. Are we cool with exposing an `is_fair_partner?` in JSON? I personally think that's fine (tho not my call I guess). In that case, we can just expose that directly, and kill `claimed` altogether.

`claimed` is already automatically updated to be: https://github.com/artsy/gravity/blob/3b8a82b8adedcc6986625ff231df322bea67bb19/app/models/domain/partner.rb#L300. It feels like a field that only exists as a 'wrapper' for `is_fair_partnership?`, which I don't think we need.